### ISSUE TITLE
New custom effect

### ISSUE THREAD

**Issue by Miesjelangelo** [bruhautomation/ESP-MQTT-JSON-Digital-LEDs#89]
Hi all,

not sure if this is the correct place for my question, but lets hope so!

I've managed to get everything working with the provided code but I'm planning on using the strip in the garden as a way of lighting up the driveway. 

If possible, i would like to add an effect that lights op a couple of leds every few feet (like spots), and when someone enters the driveway I would like it to light up the whole strip (one at a time from front to backup and than stay on). 

I've tried learning from the code in the arduino file but could not create an effect of my own. 

Michel

**Comment by federom** [comment]
Yes you can if you connect this to Home Assistant, and then you create an automation there to control the strip in the way you described

**Comment by Miesjelangelo** [comment]
Thanks for responding! 

But I can't controle each individual led via home assistent, i can however tell it to start with a specific effect. Or am I wrong? 

That's why I said, I need to create a new effect.

**Comment by federom** [comment]
i would create an effect in the .ino file and then i would call it from an HA automation

**Comment by Joeboyc2** [comment]
That's the point miesjel12 is making, he/she wants help in making the effect

**Comment by Miesjelangelo** [comment]
Exactly, thanks 😁
